views/dashboard/about-page/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react'
+import { useSelector } from 'react-redux'
+import FormView from './components/FormView'
+import { StyledAboutWidget } from './StyledAboutWidget'
+
+const AboutWidget = () => {
+    const AboutPageData = useSelector((state: any) => state.aboutPage)
+    const [formView, setFormView] = useState<boolean>(true)
+
+    const handleViewChange = () => {
+        setFormView(!formView)
+    }
+
+    return (
+        <StyledAboutWidget>
+            <div className="header">
+                <h3
+                    className={`${formView && 'active'}`}
+                    onClick={handleViewChange}>
+                    ABOUT ME
+                </h3>
+                <span>/</span>
+                <h3
+                    className={`${!formView && 'active'}`}
+                    onClick={handleViewChange}>
+                    COMPANY STORE
+                </h3>
+            </div>
+            <div className="form-view">
+                {formView ? (
+                    <FormView FormData={AboutPageData?.[0]} />
```

Comment:
Also, we only ever want to capitalize prop variables if they are rendered elements like Icon. CamelCase my guy. First letter lowercase, every other first word first letter upper case.

Reply:
I CamelCased how you wrote it 😂🤣